prow/config.yaml

```diff
@@ -1697,7 +1697,7 @@ presubmits:
         - "--repo=k8s.io/release"
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - "--timeout=90"
+        - "--timeout=110"
```

Comment:
I don't understand why this timeout is so much longer than the others.

Reply:
What this timeout actually is? Why do we have 3 different types of timeout?

[I understand two: for tests themself, and then longer for really kill everything]

BTW @BenTheElder @ixdy - can we merge this today - without it nothing is merging to 1.7 branch even.

Reply:
- jenkins has a timeout concept
- bootstrap (effectively job init script) has a timeout concept
- kubetest has a timeout concept

We have to bump jenkins because 1.6 isn't dead yet
I bumped bootstrap out of habit, this one is pretty is just a sanity check to make sure the job doesn't hang. It will probably be moved into Prow
Kubetest is the one we're hitting, and needs to be lower than the other two.